packages/admin/src/pages/build-application/[applicationId]/components/Sections.tsx

```diff
@@ -36,27 +45,49 @@ const Sections = ({
               <Table
                 caption={
                   sectionIndex >= 2 && applicationStatus != 'PUBLISHED' ? (
-                    <div className="govuk-width-container">
-                      <div className="govuk-grid-row">
-                        <div className="govuk-grid-column-one-half">
-                          {sectionIndex + 1}. {section.sectionTitle}
-                        </div>
-                        <div className="govuk-grid-column-one-half">
-                          <p className="govuk-!-text-align-right govuk-!-font-size-19 govuk-!-margin-0">
-                            <CustomLink
-                              href={`/build-application/${applicationId}/${section.sectionId}`}
-                            >
-                              Edit section
-                            </CustomLink>
-                          </p>
-                        </div>
+                    <div className="govuk-grid-row govuk-!-padding-top-5 govuk-!-padding-bottom-3 govuk-!-padding-left-3 govuk-!-padding-right-3">
+                      <div className="govuk-grid-column-one-third">
+                        {sectionIndex + 1}. {section.sectionTitle}
+                      </div>
+
+                      <div className="govuk-grid-column-two-thirds govuk-!-text-align-right">
+                        <button
+                          className={`govuk-button govuk-!-margin-right-2 govuk-!-margin-bottom-0 ${styles['b']}`}
+                          data-module="govuk-button"
+                          data-cy="cyUpButton"
+                          name={`Up/${section.sectionId}`}
+                          disabled={sectionIndex === 2}
```

Comment:
Can again extract this random "2" to a well named variable. There's a couple places this could be used

Reply:
Fixed in latest commit